tmk_core/common/command.c

```diff
@@ -358,7 +358,7 @@ static bool command_common(uint8_t code)
             print("\n\nJumping to bootloader... ");
             #ifdef AUDIO_ENABLE
 	            stop_all_notes();
-                play_goodbye_tone();
+                goodbye_song_user();
```

Comment:
I'm not entirely sure this should be a "song" - what if we just called it `goodbye_user` here, to let people flash leds, play notes, or send keystrokes?